modules/samba/samba.json

```diff
@@ -7,17 +7,15 @@
         "./buildtools/bin/waf install"
     ],
     "cleanup": [
-        "/bin",
```

Comment:
If `nmblookup` isn't needed then `cleanup`  should stay untouched since there are tins of unnecessary stuff there.

Reply:
Whoops, forgot, that these were multiple commits